csv.mk

```diff
@@ -8,15 +8,15 @@ unzip -p $< > $@
 endef
 
 %.zip :
-	wget http://www.isbe.net/research/zip/$@
+	wget https://www.isbe.net/_layouts/Download.aspx?SourceUrl=/Documents/$@ -O $@
 
-.INTERMEDIATE : rc11.zip rc12.zip rc13.zip rc14.zip
-rc11.zip rc12.zip rc13.zip rc14.zip :
-	wget http://www.isbe.net/assessment/zip/$@
+.INTERMEDIATE : rc11.zip rc12.zip rc13.zip rc14.zip rc15.zip
+rc11.zip rc12.zip rc13.zip rc14.zip rc15.zip:
```

Comment:
looks like this rule is no longer necessary as it would now be covered by the rule on line 11?